### DISCUSSION TITLE
Copy Annotations Guidance

### DISCUSSION THREAD

**Discussion by briandowns** [Jason2605/Dictu#653] [General] [answered]
I've done a lot of experimenting and can't seem to find the best place to perform the copy from super class to class for annotations. Any guidance here would be appreciated. Thanks!

**Comment by Jason2605** [comment, marked as answer]
I haven't had a massive look into it myself yet, but my initial attempt would be to copy the annotation dict from the parent here:
- https://github.com/dictu-lang/Dictu/blob/develop/src/vm/vm.c#L761-L764

And then when the annotations are actually getting defined we'd need to see if we need to extend an existing dict or set the annotations in here:
- https://github.com/dictu-lang/Dictu/blob/develop/src/vm/vm.c#L2186-L2211

**Reply by briandowns** [reply]
Ok, I was on the right path. I'll keep going on this then. Thanks!

**Reply by Jason2605** [reply]
Nice one thank you!
